site/content/how-to/traffic-management/backend-tls-termination.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Backend TLS Termination"
```

Comment:
I was going to mention that "termination" feels like a weird term here, since we're initiating TLS, not terminating it, but the GEP does use that terminology...

Reply:
Yeah, I agree, it feels weird to me too, but I guess it's worded that way _because_ the backend is the one doing the termination?